### DISCUSSION TITLE
проблема при компиляции

### DISCUSSION THREAD

**Discussion by akomylator** [vvip-68/LedPanelWiFi#85] [Вопросы и ответы]
exit status 1 'sntp_stop' was not declared in this scope; did you mean 'igmp_stop'?
не компилируеться скейтч

**Comment by vvip-68** [comment]
Ядро esp8266 - 3.1.2, esp32 - 2.0.14
Библиотеки - из папки проекта

**Reply by akomylator** [reply]
ядро 3.0.7

**Reply by vvip-68** [reply]
Сочувствую

**Reply by akomylator** [reply]
я так понимаю работает только на 2.0.14?

**Reply by vvip-68** [reply]
Именно
